Automattic/pocket-casts-ios
# watchOS 11: make play/pause button a primary action

While building the project for Xcode 16 I went ahead and implemented the primary action for the play/pause button which should allow to play/pause by double-tapping the thumb and index finger together.

Before merging that we need Xcode 16.0 on the CI though, so I'll leave as a draft till the support is added.

## To test

1. Run the app on a device connected to a Apple Watch Series 9 and Apple Watch Ultra 2 with watchOS 11
2. Open the playing screen on the watch
3. Double-tap using the thumb and index finger
4. ✅ Episode should play/pause

## Checklist
